### PR TITLE
Update readme for Leap's collection of examples page

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Added content from docs, added an installation section, fixed a long-missed naming bug (I don't know how we never spotted this one).

### CODE REVIEW

File: 01-exploring-pegasus.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "\n",
     "The Advantage system is distinct from all previous generations of D-Wave's quantum computers in being the first to employ a new architecture made possible by recent advances in QPU fabrication technology. \n",
     "\n",
-    "QPUs are lattices of interconnected qubits. While some qubits connect to others via couplers, QPUs are not fully connected. Instead, qubits interconnect in a topology: Chimera for the DW-2000Q and Pegasus for the Advantage.\n",
+    "QPUs are lattices of interconnected qubits. While some qubits connect to others via couplers, QPUs are not fully connected. Instead, qubits interconnect in a topology: Chimera for the D-Wave 2000Q and Pegasus for the Advantage.\n",
```
Comment (randomir) [inline 01-exploring-pegasus.ipynb:34]: Mention Zephyr/Advantage2?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,47 @@ unit (QPU) with a new architecture: the Pegasus family of topologies. This
...
+QPU, you should understand it if you are using QPU solvers directly because it
+has implications for the problem-graph size and solution quality.
+
+The D-Wave QPU is a lattice of interconnected qubits.
+While some qubits connect to others via couplers, the D-Wave QPU is not fully
+connected. Instead, the qubits of D-Wave annealing quantum computers interconnect
+in one of the following topologies:
+
+* Chimera for D-Wave 2000Q and earlier generations of QPUs
+* Pegasus for Advantage QPUs
```
Comment (randomir) [inline README.md:36]: Zephyr?

**Review by randomir — APPROVED** [review]
LGTM

File: 01-exploring-pegasus.ipynb (hunk 1 above)
Comment (JoelPasvolsky) [reply 01-exploring-pegasus.ipynb:34]: I think not quite yet. Let Pegasus have its full day in the sun.

**Pull request merged**